README.md

```diff
@@ -10,3 +9,33 @@ For detailed API description, please see [the draft on the project
 wiki](https://github.com/codingteam/Gestalt/wiki/Public-API).
 
 For copying terms, see LICENSE file.
+
+# Buildig and running the service
+
+You need the following installed in your system:
+
+* [Erlang version 17 or above](erlang.org)
+* [rebar](https://github.com/rebar/rebar/wiki/Getting-started#first-steps)
+
+Now run:
+
+    # compile the source and generate the release
+    $ rebar compile generate
+
+    # start the service
+    $ ./rel/gestalt/bin/gestalt start
```

Comment:
Both of your suggestions worked, thank you. BTW, I've edited my initial comment and already said that `gestalt install` worked. Thank you anyway.